crates/wysiwyg/src/tests/test_paragraphs.rs

```diff
@@ -429,3 +429,35 @@ fn pressing_enter_at_the_start_of_a_multiline_block_quote() {
         "<blockquote><p>&nbsp;</p><p>|line_1</p><p>line_2</p></blockquote>"
     )
 }
+
+#[test]
+fn pressing_enter_in_the_middle_of_a_multiline_code_block_with_empty_starting_paragraphs(
+) {
+    let mut model = cm("<pre>|line_1\nline_2</pre>");
+    model.enter();
+    assert_eq!(tx(&model), "<pre>&nbsp;\n\n|line_1\nline_2</pre>")
```

Comment:
this one feels a bit weird... look like two lines have been added?
The thing is that, what should be a test case of pressing enter in a situation like this?
![simulator_screenshot_C4F662C8-6EC4-47DC-8333-F495A1C3E62E](https://user-images.githubusercontent.com/34335419/215087116-166dadc5-5623-4153-8afa-d15b413e6de3.png)

Reply:
Actually, that one was broken, too much copy&paste 😓 . It should be fixed in [`63969e9`](https://github.com/matrix-org/matrix-rich-text-editor/pull/531/commits/63969e9ebd3eb3ad38f7cee19f990f001b388fe6), adding a single paragraph before the starting one.